_layouts/team-member.html

```diff
@@ -42,12 +42,46 @@ <h1 class="title">{{ page.firstname }} {{ page.lastname }}
                             {{ link.name }}
                         </a>
                     </li>
-                    {% endfor %}
+                    {% endfor %}    
                 </ul>
             </div>
             {% endif %}
         </div>
     </div>
+
+    <!-- blog posts -->
+    <div class="row">
+        <div class="col">
```

Comment:
Is this supposed to be `.row > .col`? There is no row or column haha

Reply:
Ah, I thought every section of a page was put into a grid and thus has a row/col. Now I know better of the structure :) thanks!